<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>800</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLabel" name="videoPreview">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>960</width>
      <height>540</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>Video Preview</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>-1</y>
      <width>301</width>
      <height>541</height>
     </rect>
    </property>
    <property name="title">
     <string>IP Camera</string>
    </property>
    <widget class="QPushButton" name="pushBtn_Video">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>490</y>
       <width>141</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Open Streams</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="camIP_0">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>30</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.142</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="camIP_1">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.142</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="camIP_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>90</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.142</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="camIP_3">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>201</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>192.168.1.142</string>
     </property>
    </widget>
    <widget class="QComboBox" name="box_protocol_1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>180</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>RTSP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>HTTPS</string>
      </property>
     </item>
    </widget>
    <widget class="QCheckBox" name="checkBox">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Processing</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>150</y>
       <width>100</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>554</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>540</y>
      <width>301</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>General Settings</string>
    </property>
    <widget class="QPushButton" name="pushBtn_Map">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>140</y>
       <width>141</width>
       <height>40</height>
      </rect>
     </property>
     <property name="text">
      <string>Display Map</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>540</y>
      <width>960</width>
      <height>190</height>
     </rect>
    </property>
    <property name="title">
     <string>Sensor Preview</string>
    </property>
    <widget class="QWidget" name="iot_widget" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>960</width>
       <height>170</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuSettings"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
